download_and_create_reference_datasets/v03/write_clinvar_ht.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import tempfile
+
+import hail as hl
+
+from hail_scripts.reference_data.constants import GCS_PREFIXES
+from hail_scripts.utils.clinvar import (
+    download_and_import_latest_clinvar_vcf,
+    parsed_clnsig,
+    parsed_clnsigconf,
+    CLINVAR_ASSERTIONS_LOOKUP,
+    CLINVAR_DEFAULT_PATHOGENICITY,
+    CLINVAR_PATHOGENICITIES_LOOKUP,
+    CLINVAR_GOLD_STARS_LOOKUP,
+)
+from hail_scripts.utils.hail_utils import write_ht
+
+CLINVAR_HT_PATH = 'clinvar/clinvar.GRCh{genome_version}.{timestamp}.ht'
+PARTITIONS = 100 # per https://github.com/broadinstitute/seqr-loading-pipelines/pull/383
+
+def run(environment: str):
+    for genome_version in ['37', '38']:
+        with tempfile.NamedTemporaryFile(delete=False, suffix='.vcf.gz') as tmp_file:
+            mt = download_and_import_latest_clinvar_vcf(genome_version, tmp_file)
+            timestamp = hl.eval(mt.version)
+            ht = mt.rows()
+            ht = ht.select(
+                alleleId=ht.info.ALLELEID,
+                pathogenicities_id=CLINVAR_PATHOGENICITIES_LOOKUP.get(
```

Comment:
Another potential option here is to filter out the assertions, then do non-safe lookup of the pathogenicity lookup so we fail if we find an unexpected one.

Reply:
I think we also do want to make sure that the pathogenicity is only at the first position in the list, I think the way this is currently written it would not throw an error if the second list item was a pathogenicity and just skip it, and I think the other potential solution you propose would also have that issue. I think maybe we can leave this as is, but then add a conditional for where we start checking the assertion list, so something like 
```
assertions_ids=hl.if_else(CLINVAR_PATHOGENICITIES_LOOKUP.contains(clinsigs[0]), clinsigs[1:], clinsigs).map(
  lambda x: CLINVAR_ASSERTIONS_LOOKUP[x]
)
```